tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -38,7 +38,7 @@ Verify Traefik Deployment
 Verify JH Deployment
     [Documentation]  Verifies RHODS JH deployment
     [Tags]    Sanity
-    ...       ODS-546  ODS-294  ODS-1250
+    ...       ODS-546  ODS-294  ODS-1250  ODS-237
```

Comment:
is ODS-1250 covered by this code? 1250 should check the JH version. 

My only doubt about ODS-237 (`Verify that Jupiterhub is shipped and enabled within RHODS`) is that other TCs similar to this one (e.g., `Verify Blackbox exported is shipped and enabled ..`) have more checks (e.g., `OpenShift Resource Field Value Should Be Equal As Strings ...`)

Reply:
I also agree that you can create a new file for all tests related to jupyterhub deployment (1xy__jupyterhub.robot). It could be done in a future PR